testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -1,22 +1,34 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
+import LoginInstitutionObject from "../../page-objects/Institution-objects/LoginInstitutionObject";
 
-describe("Login Page", () => {
-  const dashboardInstitutionObject = new DashboardInstitutionObject();
-  const institutionCustomCommand = new InstitutionCustomCommand();
+const dashboardObject = new DashboardInstitutionObject();
+const iCC = new InstitutionCustomCommand();
+const loginObject = new LoginInstitutionObject();
 
-  const url = Cypress.env("institutionURL");
+const LOGIN_URL = Cypress.env("TEST").BASE_URL + "/institution/login";
+const UNAME = Cypress.env("TEST").UNAME_1;
+const PASS = Cypress.env("TEST").PASS_1;
 
-  before(() => {
-    cy.visit(url);
+describe("Login Page", () => {
+  beforeEach(() => {
+    cy.visit(LOGIN_URL);
   });
 
-  it("Verify login case in institution.", () => {
-    institutionCustomCommand.loginInstitution();
+  it("Verify invalid uname/password field validation error", () => {
```

Comment:
`username/password` ?

Reply:
Just to validate that there is an error popping up during during "incorrect" username or password

Reply:
mm, my comment is to change `uname`  to `username`